R/run_eggla_gwas.R

```diff
@@ -528,8 +574,8 @@ run_eggla_gwas <- function(
           USE.NAMES = FALSE
         )
 
-        info_fields_required <- c("NS", "AC_Het", "AC_Hom", "AF", "HWE", "F_MISSING", "INFO", "R2")
-        names(info_fields_required) <- c("N", "N1", "N2", "EAF", "HWE_P", "CALL_RATE", "INFO", "INFO")
+        info_fields_required <- c("INFO", "R2")
+        names(info_fields_required) <- c("INFO", "INFO")
```

Comment:
Now, we only extract imputation related fields from VCF files.

Reply:
Sorry I think I clicked resolve too quickly. I can't confirm if F_MISSING is really the CALL_RATE?

Reply:
Not exactly, it's 1 - F_MISS = CALL_RATE.
I output CALL_RATE instead of F_MISS.

Reply:
I am resolving this thread, because it's not where the F_MISSING/CALL_RATE really is.
This correspondence was for BCFtools.